prime-router/src/main/kotlin/Mappers.kt

```diff
@@ -225,16 +225,23 @@ class LIVDLookupMapper : Mapper {
             error("Schema Error: livdLookup mapper does not expect args")
         // EQUIPMENT_MODEL_NAME is the more stable id so it goes first. Device_id will change as devices change from
         // emergency use to fully authorized status in the LIVD table
-        return listOf(EQUIPMENT_MODEL_NAME, DEVICE_ID, EQUIPMENT_MODEL_ID, TEST_KIT_NAME_ID)
+        return listOf(EQUIPMENT_MODEL_NAME, DEVICE_ID, EQUIPMENT_MODEL_ID, TEST_KIT_NAME_ID, TEST_PERFORMED_CODE)
     }
 
     override fun apply(element: Element, args: List<String>, values: List<ElementAndValue>): String? {
+        // get the test performed code
+        val testPerformedCode = values.firstOrNull { it.element.name == TEST_PERFORMED_CODE }?.value
+        val filters: MutableMap<String, String> = mutableMapOf()
+        if (!testPerformedCode.isNullOrEmpty()) {
+            filters[LIVD_TEST_PERFORMED_CODE] = testPerformedCode
+        }
```

Comment:
Get the `test_performed_code` and create a map of filter values for this table to work with